server/svix-server/src/worker.rs

```diff
@@ -334,21 +334,32 @@ async fn dispatch(
             };
             let http_error = res.error_for_status_ref().err();
 
-            let bytes = res
-                .bytes()
-                .await
-                .expect("Could not read endpoint response body");
-            let body = bytes_to_string(bytes);
+            match res.bytes().await {
```

Comment:
It's not clear to me that this should be treated as a failed attempt. Presumably we got back a successful status, but for whatever reason reading the body timed out. Why not log/record the failure reading the body but treat it as a success? Otherwise, we will surely be sending duplicate messages when we retry.